daisy/project.go

```diff
@@ -37,7 +37,7 @@ func projectExists(client compute.Client, project string) (bool, error) {
 		if apiErr, ok := err.(*googleapi.Error); ok && apiErr.Code == http.StatusNotFound {
 			return false, nil
 		}
-		return false, err
+		return false, typedErr(apiError, err)
```

Comment:
My issue with this is you wont be able to check the api error in calling functions. You don't need to change this PR but maybe we should just put the API error code as the type? Or something like that so we don't have to have a daisy error type for every possible api error we get but can also still check (or probably just print out) what it is.